libs/ui-watt/src/lib/components/modal/watt-modal.component.ts

```diff
@@ -80,6 +80,10 @@ export class WattModalComponent implements AfterViewInit {
   @Input()
   loading = false;
 
+  /** Whether the modal should show a loading text for the loading state. */
+  @Input()
+  loadingMessage = '';
```

Comment:
Are there no tests associated with this component? (They should properly be expanded to have a few tests for this loading message)